tests/test_data/kitchendreaming.com/kitchendreaming_1.json

```diff
@@ -0,0 +1,84 @@
+{
+  "author": "Ronda Eagle | Kitchen Dreaming",
+  "canonical_url": "https://kitchendreaming.com/crock-pot-buffalo-chicken-meatballs/",
+  "site_name": "KitchenDreaming",
+  "host": "kitchendreaming.com",
+  "language": "en-US",
+  "title": "Crock Pot Buffalo Chicken Meatballs",
+  "ingredients": [
+    "1-1/2 pounds ground chicken ([See Note 1])",
+    "1/2 cup panko bread crumbs ( [See Note 2])",
```

Comment:
For a moment I was wondering whether to suggest attempting to filter out these `See Note` statements -- we don't include the accompanying notes in the scraper field output, as far as I know.  However: I think we should keep them.  Removing them would mean that users would be entirely unaware that there is additional (and presumably, somewhat relevant -- if it is called out in the ingredient list) context about them.

They may have to seek that information out themselves if they want/need to know it - but I think that's a better tradeoff than hiding it entirely.

(in a philosophical sense someone might argue that what scrapers do in their entirety is to remove surrounding context -- I'd argue that there's some truth in that, but that it's possible to make judgement calls about what is relevant and what is not)